doc/examples/animations-pulsating-sphere.ipynb

```diff
@@ -0,0 +1,341 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Animations of a Pulsating Sphere"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sfs\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import animations_pulsating_sphere as ani\n",
+    "from IPython.display import display, Image, HTML"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Pulsating sphere\n",
+    "center = [0, 0, 0]\n",
+    "radius = 0.25\n",
+    "amplitude = 0.05\n",
+    "f = 1000  # frequency\n",
+    "omega = 2 * np.pi * f  # angular frequency\n",
+    "\n",
+    "# Axis limits\n",
+    "figsize = (8, 8)\n",
+    "xmin, xmax = -1, 1\n",
+    "ymin, ymax = -1, 1\n",
+    "\n",
+    "# Animations\n",
+    "frames = 20  # frames per period"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Particle Displacement"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grid = sfs.util.xyz_grid([xmin, xmax], [ymin, ymax], 0, spacing=0.025)\n",
+    "ani_obj = ani.animation_particle_displacement(\n",
+    "            omega, center, radius, amplitude, grid, frames, figsize, c='Gray')\n",
+    "ani_obj.save('pulsating_sphere_displacement_square_grid.gif',\n",
+    "             dpi=80, writer='imagemagick')\n",
+    "plt.close()\n",
+    "Image('pulsating_sphere_displacement_square_grid.gif')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can play with the animation more interactively by using `.to_jshtml`.\n",
+    "The playback speed and direction can be changed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "display(HTML(ani_obj.to_jshtml(default_mode='loop')))"
```

Comment:
No need to use `display()` here.

Reply:
Also, `'loop'` seems to be the default, no need to specify it here.

Reply:
> I think this would look nicer, what do you think?

Yeah, good idea!